src/main/java/eatwhere/foodguide/model/eatery/NameContainsKeywordsPredicate.java

```diff
@@ -18,7 +19,8 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Eatery eatery) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(eatery.getName().fullName, keyword));
+                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(eatery.getName().fullName
+                        .replaceAll("\\pP", ""), keyword));
```

Comment:
```suggestion
                        .replaceAll("\\pP", ""), keyword.replaceAll("\\pP", "")));
```
Currently find non-aircon and find hwang's returns 0 eateries, due to punctuation being removed for the eatery name but not the keyword. But this will also introduce somewhat weird results like find hwang's returning hwangs